docker-compose.yml

```diff
@@ -274,38 +274,21 @@ services:
         aliases:
           - backend.corporanet.local
   oidc:
-    image: soluto/oidc-server-mock:0.3.0
-    # For Mac M1, you may need to build this image from source.
-    # 1. Comment out `image:` line, above, and uncomment the following two lines
-    # 2. In the same directory as this file, run `git clone -b v0.3.0 git@github.com:Soluto/oidc-server-mock.git`
-    # container_name: oidc-server-mock-local
-    # build: oidc-server-mock/src
+    build:
+      cache_from:
+        - "scdp-pkce-auth"
+      context: oauth
+      dockerfile: Dockerfile
+    restart: always
     ports:
-      - "4011:80"
-      - "8443:8443"
-    environment:
-      ASPNETCORE_ENVIRONMENT: Development
-      SERVER_OPTIONS_INLINE: |
-        {
-          "AccessTokenJwtType": "JWT",
-          "Discovery": {
-            "ShowKeySet": true
-          }
-        }
-      API_SCOPES_PATH: /tmp/config/api_scopes.json
-      API_RESOURCES_PATH: /tmp/config/api_resources.json
-      USERS_CONFIGURATION_PATH: /tmp/config/users.json
-      CLIENTS_CONFIGURATION_PATH: /tmp/config/clients-config.json
-      ASPNETCORE_URLS: https://+:8443;http://+:80
-      # ASPNETCORE_Kestrel__Certificates__Default__Password: <password for pfx file>
-      ASPNETCORE_Kestrel__Certificates__Default__Path: /tmp/pkcs12/certificate.pfx
+      - "443:443"
     volumes:
-      - ./oauth/pkcs12:/tmp/pkcs12:ro
-      - ./oauth:/tmp/config:ro
+      - ./oauth/pkcs12:/tmp/pkcs12/:ro
     networks:
       corporanet:
         aliases:
           - oidc.corporanet.local
```

Comment:
I'm leaving this as `oidc` for now because I could not get it to work otherwise. Some issue with local Docker networking configuration, surely. I tried `docker network prune` and confirmed that the `corporanet` network had to be remade; tried clearing DNS caches; nothing worked.

Reply:
My first thought, reading the above, would be that I made a mistake and that the original oidc container was still running, etc. etc., and I was authenticating with that 😂 

This is not the case.